core/conf/config.go

```diff
@@ -63,6 +63,11 @@ type Config struct {
 
 	// Ethereum node: "https://ropsten.infura.io/v3/xxxxxxxxxxxxxxx"
 	EthereumNode string `json:"ethereum_node,omitempty"`
+
+	// ZboxApi 0box api host host: "https://0box.dev.0chain.net"
+	ZboxApi string `json:"zbox_api"`
```

Comment:
If this is service host address, then let's name it `ZboxHost` or `ZboxAddr`

Reply:
> We can use zbox as package name, the suffix `api` is not necessary as all the stuff in the gosdk are SDKs.

we already have a `zbox` package. it is a wrapper of zbox cli. `zboxapi` is different from it. it is client sdk of github.com/0chain/0box.  
do you think they should be merged into one package?

Reply:
@peterlimg  updated it with `ZboxHost`

Reply:
okay, then let's keep it as `zboxapi`